locales/ro_RO/LC_MESSAGES/duckduckgo.po

```diff
@@ -3713,16 +3724,20 @@ msgid "Turn off:"
 msgstr "Dezactivează:"
 
 msgid "TV"
-msgstr ""
+msgstr "TV"
 
 msgid "Twitter"
-msgstr ""
+msgstr "Twitter"
 
+# Ambiguous
+#, fuzzy
 msgid "Type %sDuckDuckGo%s in the first %sform field"
-msgstr "Tastează %sDuckDuckGo%s în primul %scâmp al formularului"
+msgstr "Scrie %sDuckDuckGo%s în primul câmp din %s"
```

Comment:
Can you please show an example for this string?

Reply:
You should be able to see this string on https://duckduckgo.com/install from Chrome or Chromium after clicking the "Don't see DuckDuckGo in the list?" link.

![chr](https://user-images.githubusercontent.com/444449/52350049-a27d8a80-2a1f-11e9-9a68-45c11b6fe9e2.png)

Reply:
Aah, I gather the last placeholder in _Type %sDuckDuckGo%s in the first **%s**form field_ is for `<br>`.

I will need to update these too